plugin/src/sbt-test/paradox/latest-paradox-deps/project/plugins.sbt

```diff
@@ -0,0 +1,8 @@
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox"       % "0.10.6")
```

Comment:
This is the important part, as you can see we are loading the most recent version of sbt-paradox/sbt-paradox-theme